engines/bladerunner/subtitles.cpp

```diff
@@ -147,12 +147,13 @@ void Subtitles::init(void) {
 		       _subtitlesInfo.languageMode.c_str(),
 		       _subtitlesInfo.credits.c_str());
 
-		delete versionTxtResource;
-		versionTxtResource = nullptr;
 	} else {
 		debug("Subtitles version info: N/A");
 	}
 
+	delete versionTxtResource;
```

Comment:
oh I missed that, thanks!
versionTxtResource can be local variable directly (not on heap), so no "new"  and "delete" are needed